ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CassJavaDriverBatch.java

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) 2017 Netflix, Inc.  All rights reserved.
+ */
+package com.netflix.ndbench.plugin.cass;
+
+import com.datastax.driver.core.*;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.netflix.archaius.api.PropertyFactory;
+import com.netflix.ndbench.api.plugin.annotations.NdBenchClientPlugin;
+import com.netflix.ndbench.api.plugin.common.NdBenchConstants;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Random;
+/**
+ * @author vchella
+ */
+@Singleton
+@NdBenchClientPlugin("CassJavaDriverBatch")
+public class CassJavaDriverBatch extends CJavaDriverBasePlugin {
+
+    //Settings
+    private static String TableName2;
+    private static Integer batchSize;
+    private static Boolean useTimeStamp;
+    private static Boolean useMultiPartition;
+
+    Random randomObj = new Random();
+
+    protected PreparedStatement writePstmt2;
+
+    @Inject
+    public CassJavaDriverBatch(PropertyFactory propertyFactory, CassJavaDriverManager javaDriverManager) {
+        super(propertyFactory, javaDriverManager);
+    }
+
+    @Override
+    void prepStatements(Session session) {
+
+        readPstmt = session.prepare(" SELECT cyclist_name, expense_id, amount, description, paid FROM "+TableName+" WHERE cyclist_name = ?" );
+
+        writePstmt = session.prepare("INSERT INTO "+TableName+" (cyclist_name, expense_id, amount, description, paid) VALUES (?, ?, ?, ?, ?)");
+        writePstmt2 = session.prepare("INSERT INTO "+TableName2+" (expense_id, cyclist_name) VALUES (?, ?)");
+
+
+    }
+
+    @Override
+    void upsertKeyspace(Session session) {
+        upsertGenereicKeyspace();
+    }
+
+    @Override
+    void upsertCF(Session session) {
+        session.execute("CREATE TABLE IF NOT EXISTS "+TableName+" (cyclist_name text, balance float STATIC, expense_id int, amount float, description text, paid boolean, PRIMARY KEY (cyclist_name, expense_id) )");
+        session.execute("CREATE TABLE IF NOT EXISTS "+TableName2+" (expense_id int, cyclist_name text, PRIMARY KEY (expense_id, cyclist_name)) ");
+    }
+
+    @Override
+    void preInit() {
+        TableName2  = propertyFactory.getProperty(NdBenchConstants.PROP_PREFIX+"cass.cfname2").asString("test2").get();
+        batchSize = propertyFactory.getProperty(NdBenchConstants.PROP_PREFIX+"cass.batchSize").asInteger(3).get();
+        useTimeStamp = propertyFactory.getProperty(NdBenchConstants.PROP_PREFIX+"cass.useTimestamp").asBoolean(true).get();
+        useMultiPartition = propertyFactory.getProperty(NdBenchConstants.PROP_PREFIX+"cass.useMultiPartition").asBoolean(false).get();
+    }
+
+    @Override
```

Comment:
`postInit()` is empty and it seems other definitions below have also not been implemented.